app/components/settings/email-preferences-section.js

```diff
@@ -3,4 +3,9 @@ import Ember from 'ember';
 const { Component } = Ember;
 
 export default Component.extend({
+  actions: {
+    save(notificationType, setting) {
```

Comment:
Why do you need send action when you can directly refer to the passed in action in the template ?